internal/provider/ovirt/ignition.go

```diff
@@ -0,0 +1,63 @@
+package ovirt
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/go-openapi/swag"
+	"github.com/openshift/assisted-service/models"
+	"gopkg.in/yaml.v2"
+)
+
+type OvirtConfig struct {
+	URL      string `yaml:"ovirt_url,omitempty"`
+	Username string `yaml:"ovirt_username,omitempty"`
+	Password string `yaml:"ovirt_password,omitempty"`
+	Insecure bool   `yaml:"ovirt_insecure,omitempty"`
+	CaBundle string `yaml:"ovirt_ca_bundle,omitempty"`
+}
+
+func (p ovirtProvider) createOvirtConfig(workDir string, platformParams *models.Platform) (string, error) {
+	if platformParams.Ovirt == nil {
+		return "", nil
+	}
+	URL := fmt.Sprintf("https://%s/ovirt-engine/api", *platformParams.Ovirt.Fqdn)
+	oVirtConfig := &OvirtConfig{
+		URL:      URL,
+		Username: swag.StringValue(platformParams.Ovirt.Username),
+		Password: platformParams.Ovirt.Password.String(),
+		Insecure: swag.BoolValue(platformParams.Ovirt.Insecure),
+		CaBundle: swag.StringValue(platformParams.Ovirt.CaBundle),
+	}
+	ovirtConfigPath := filepath.Join(workDir, "ovirt-config.yaml")
```

Comment:
where is `workDir`? I wonder if it is safe to write to it.
Anyway write the file with ".ovirt-config.yaml"